README.md

```diff
@@ -109,6 +109,55 @@ A glossary entry is structured like this:
     should `define('something')` in RStudio put the definition in the help pane
     (and if so, should it hyperlink to terms that the definition depends on)?
 
+## Use Cases
+
+1.  **Linking to a definition.**
+    1.  *Amari* writes a lesson in R Markdown that introduces some new terms.
+    1.  She has defined the language to be Spanish using the `glossary/language` key
+        in the YAML header,
+        but has not changed any other settings.
+    1.  She adds an inline code block `` `r gdef('linear-model', 'Linear models')` ``
+        to her lesson.
+    1.  When she knits her document,
+        the inline code block produces the HTML
+        `<a href="http://carpentries.org/glossary/es/#linear-model" class="glossary-definition">Linear Models</a>`
+
+1.  **Checking a lesson.**
```

Comment:
I strongly feel this shouldn't be a use case and that it instead should be part of the packages.

Reply:
Sorry, it should be a use case but wrapped under a function.

Reply:
I don't understand - as written, it *is* a function call. Can you give an example of what the user would type in their R Markdown?

Reply:
We cleared this up in a conversation/call. As it is written it is now clear.